tests/testthat/test-filter_counts.R

```diff
@@ -1,13 +1,20 @@
 set.seed(1665)
-sce <- sim_sce(n_cells = 100, n_genes = 200, n_empty = 1000)
+sce <- sim_sce(n_cells = 100, n_genes = 200, n_empty = 100)
```

Comment:
Did you intend to reduce the # empty? At first I didn't expect it would matter but I would expect a larger number is more realistic for estimation of ambient profile. I am noted that the default for `DropletUtils:::simCounts()` is 10,000 empty cells, which could make a difference in the estimation of the empty cells. 

Update:
Looking at the docs for `[emptyDropsCellRanger](https://rdrr.io/github/MarioniLab/DropletUtils/man/emptyDropsCellRanger.html)`:
>emptyDropsCellRanger takes barcodes whose total count ranks within a certain range - by default, (45,000, 90,000] - to compute the ambient profile.

Even with 10,000 empty droplets, as `simCounts()`, it would seem like there would be not ambient cells by that definition, but I still wonder if that is part of the problem. 

In any even, I would try increasing `n_empty`, and possibly `n_cells` as well to see if that makes a difference. If that doesn't work, you might be able to get somewhere with adding some of `emptyDropsCellRanger` options like `ind.min` and `ind.max` to specify which ranks are "ambient".

Reply:
Confirming my suspicion: I just did a test with `n_empty = 100000` and `emptyDropsCellRanger` worked!

Reply:
I had been playing around with both `n_empty` and `n_cells`, but I guess had not gone high enough! I'll update it now so that it has numbers where it works. Thanks for finding this!